services/auth/oauth2.go

```diff
@@ -48,6 +97,72 @@ func CheckOAuthAccessToken(accessToken string) int64 {
 	return grant.UserID
 }
 
+// GetPublicKey function to get the public key from json file
+func GetPublicKey(kid string) (interface{}, error) {
+	_, err := setting.CheckCognitoSecretFile()
+	if os.IsNotExist(err) {
+		log.Error("JSON FILE NOT FOUND %s", err)
```

Comment:
We get enough to print a warning or Debug or else less than error layer, as in first time to generate a json file in this case not have any error